app/experimenter/experiments/models.py

```diff
@@ -147,11 +155,11 @@ class Experiment(ExperimentConstants, models.Model):
     locales = models.ManyToManyField(Locale, blank=True)
     countries = models.ManyToManyField(Country, blank=True)
     projects = models.ManyToManyField(Project, blank=True)
-    platform = models.CharField(
-        max_length=255,
-        choices=ExperimentConstants.PLATFORM_CHOICES,
-        default=ExperimentConstants.PLATFORM_ALL,
+    platforms = ArrayField(
+        models.CharField(max_length=200),
+        blank=True,
         null=True,
+        default=default_all_platforms,
```

Comment:
Does this have to be a callable?  You can't just put the array literal?

Reply:
Yes this has to be a callable. Django constantly warns you about this if you don't make it a callable. Also in the docs, there is this: "If you give the field a default, ensure it’s a callable such as list (for an empty default) or a callable that returns a list (such as a function). Incorrectly using default=[] creates a mutable default that is shared between all instances of ArrayField."

Reply:
Oh right right, becuase python mutability, yep got it.  Okay then yeah leave it as a callable but just move the list of the defaults onto constantst and then you can just write a lambda that returns the constant?

Reply:
Yes, I tried that actually, however if you try to do something in the model field like:

`default=lambda: ExperimentConstants.PLATFORMS_LIST` 

you get another error when trying to generate the migration about not being able to serialize the lambda function. I believe the way I have done it is the only way to make it work.